decentraland/unity-renderer
# feat: fade out notifications

## What does this PR change?

Fix #3956 
Fades out notifications when not overing and no new notifications arrive, moving loading spinner below nearby, moving up notifications in the notifications area

...

## How to test the changes?



1. Go to: https://play.decentraland.zone/?explorer-branch=feat/notification-area-fade
2. Check that notifications appear and disappear with a fade effect
3. Fade in is faster
4. Fade out is slower

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
